File: IceCMS-front-nuxt3/types/setting.ts
export interface Setting {
    readonly id: string; // 唯一标识符，假设是字符串类型
    banquan: string; // 版权信息
    beian: string; // 网站备案号
    comment_show: boolean; // 是否显示评论
    h5Show: boolean; // 是否显示 H5 页面
    imageFormat: boolean; // 是否支持图片格式
    sitLogo: string; // 网站 logo 的 URL
    sitSrc: string; // 网站源
    sitTitle: string; // 网站标题
  }

File: IceCMS-ment/src/main/java/com/ttice/icewkment/entity/Setting.java
package com.ttice.icewkment.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.io.Serializable;

/**
 * @author admin
 * @since 2022-01-13
 */
@Data
@EqualsAndHashCode(callSuper = false)
public class Setting implements Serializable {

  /** 唯一标识符 */
  private static final long serialVersionUID = 1L;

  /** id */
  private Integer id;

  /** 备案号 */
  private String beian;

  /** 版权 */
  private String banquan;

  /** 是否展示评论 */
  private boolean comment_show;

  /** 网站标题 */
  private String sitTitle;

  /** 网站地址 */
  private String sitSrc;

  /** 网站Logo */
  private String sitLogo;

  /** 网站图片 */
  private boolean imageFormat;

  /** 移动端展示 */
  private boolean h5Show;

}
